vips/vip-400/bscmainnet.ts

```diff
@@ -0,0 +1,344 @@
+    initialSupply: {
+      amount: parseUnits("4.43236349753311919", 18),
+      vTokenReceiver: "0x46cba1e9b1e5db32da28428f2fb85587bcb785e7",
+    },
+    interestRateModel: {
+      address: "0x7DE84548C2BaDC047C5e7F0B7f9a4ba660d10dAD",
+      base: "0",
+      multiplier: "0.09",
+      jump: "0.75",
+      kink: "0.45",
+    },
+  },
+  ETH: {
+    vToken: {
+      address: ETH_VTOKEN,
+      name: "Venus ETH (Liquid Staked ETH)",
+      symbol: "vETH_LiquidStakedETH",
+      underlying: {
+        address: ETH,
+        decimals: 18,
+        symbol: "ETH",
+      },
+      ...commonSpec,
+    },
+    riskParameters: {
+      collateralFactor: parseUnits("0", 18),
+      liquidationThreshold: parseUnits("0", 18),
+      supplyCap: parseUnits("450", 18),
+      borrowCap: parseUnits("400", 18),
+      reserveFactor: parseUnits("0.15", 18),
+      protocolSeizeShare: parseUnits("0.01", 18),
+    },
+    initialSupply: {
+      amount: parseUnits("2", 18),
+      vTokenReceiver: TREASURY,
+    },
+    interestRateModel: {
+      address: "0xf03DAB984aCC5761df5f71Cc67fEA8F185f578fd",
+      base: "0",
+      multiplier: "0.035",
+      jump: "0.8",
+      kink: "0.8",
+    },
+  },
+};
+
+export const converterBaseAssets = {
+  [RISK_FUND_CONVERTER]: USDT,
+  [USDT_PRIME_CONVERTER]: USDT,
+  [USDC_PRIME_CONVERTER]: USDC,
+  [BTCB_PRIME_CONVERTER]: BTCB,
+  [ETH_PRIME_CONVERTER]: ETH,
+  [XVS_VAULT_CONVERTER]: XVS,
+};
+
+enum ConversionAccessibility {
+  NONE = 0,
+  ALL = 1,
+  ONLY_FOR_CONVERTERS = 2,
+  ONLY_FOR_USERS = 3,
+}
+
+const vip400 = (overrides: {
+  chainlinkStalePeriod?: number;
+  redstoneStalePeriod?: number;
+  hardcodeWstETHPrice?: boolean;
+}) => {
+  const meta = {
+    version: "v2",
+    title: "VIP-400",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  const chainlinkStalePeriod = overrides.chainlinkStalePeriod || CHAINLINK_STALE_PERIOD;
+  const redstoneStalePeriod = overrides.redstoneStalePeriod || REDSTONE_STALE_PERIOD;
+
+  return makeProposal(
+    [
+      // Oracle config
+      {
+        target: CHAINLINK_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[WEETH, WEETH_CHAINLINK_FEED, chainlinkStalePeriod]],
+      },
+      {
+        target: REDSTONE_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[WEETH, WEETH_REDSTONE_FEED, redstoneStalePeriod]],
+      },
+      (() => {
+        // wstETH price feed reverts in the simulation environment due to staleness check,
+        // so we have to use a stub value for testing
+        if (overrides.hardcodeWstETHPrice) {
+          return {
+            target: REDSTONE_ORACLE,
+            signature: "setDirectPrice(address,uint256)",
```

Comment:
Could we set this in the simulation?

Reply:
[a38fa42](https://github.com/VenusProtocol/vips/pull/373/commits/a38fa42eecbe361e565f23c248850fa859759ecb)